app/lib/frontend/templates.dart

```diff
@@ -849,6 +852,20 @@ String _getAuthorsHtml(List<String> authors, {bool clickableName: false}) {
   }).join('<br/>');
 }
 
+String _formatScore(double value) {
+  if (value == null) return '--';
```

Comment:
Isn't it a bug in the code if the score is not within `[0, 1]`?

Reply:
I'd have the '--' as an indicator for packages that have no analysis (e.g. because they were uploaded recently). Out of range of `[0, 1]` would be a bug, yes.